### ISSUE TITLE
MODEL SELECTION

### ISSUE THREAD

**Issue by harshan299** [IBM-EPBL/IBM-Project-30829-1660190740#1]
The MNIST training set is composed of 30,000 patterns from SD-3 and 30,000 patterns from SD-1. test set was composed of 5,000 patterns from SD-3 and 5,000 patterns from SD-1. The 60,000 pattern training set contained examples from approximately 250 writers. The sets of the training set and test set were disjoint.
